src/tests/gtests.cpp

```diff
@@ -174,3 +174,74 @@ TEST_F(LeelaTest, MoveOnOccupiedSq) {
     // Find this error in the output
     EXPECT_NE(output.find("illegal move"), std::string::npos);
 }
+
+
+TEST_F(LeelaTest, TimeControl) {
+    // Initialize network
+    cfg_weightsfile = "../src/tests/0k.txt";
+    Network::initialize();
+    cfg_max_playouts = 1;
+    cfg_num_threads = 1;
+    std::string output;
+
+    // TODO I tried to make these separate tests,
+    // but it died locally when leelaz ran the second time.
```

Comment:
Might want to figure out what's up there or we risk not being able to add any other tests...

Reply:
Fixed, but see main comments.